waku/v2/node/waku_node.nim

```diff
@@ -142,7 +142,8 @@ proc new*(T: type WakuNode, nodeKey: crypto.PrivateKey,
     nameResolver: NameResolver = nil,
     sendSignedPeerRecord = false,
     dns4DomainName = none(string),
-    discv5UdpPort = none(Port)
+    discv5UdpPort = none(Port),
+    agentString = none(string)
```

Comment:
Makes sense in principle, but afaict it would have to be something like `" "` (with a whitespace). According to me nim-libp2p would use the default agent version if supplied an empty string: https://github.com/status-im/nim-libp2p/blob/unstable/libp2p/protocols/identify.nim#L99-L100